portfolio/src/main/webapp/portfolio.html

```diff
@@ -19,7 +19,6 @@
         </ul>
     </nav>
     <header class="card px-5 py-5 my-5 text-center">
-       
```

Comment:
This commit is called "adding cards to portoflio page layout" but all it does is removing an empty line. Did you miss something?

Reply:
This must be a result of the reverting commits and having to add files again. I didn't realize it had this effect. I will make sure to be much more careful about it going forward.